Classes/Issues/Managing/IssueManagingModel.swift

```diff
@@ -11,22 +11,26 @@ import IGListKit
 
 final class IssueManagingModel: ListDiffable {
 
-    let expanded: Bool
+    let objectId: String
+    let pullRequest: Bool
 
-    init(expanded: Bool) {
-        self.expanded = expanded
+    init(objectId: String, pullRequest: Bool) {
+        self.objectId = objectId
+        self.pullRequest = pullRequest
     }
 
     // MARK: ListDiffable
 
     func diffIdentifier() -> NSObjectProtocol {
-        return "managing-model" as NSObjectProtocol
+        // should only ever be one
+        return "managing_model" as NSObjectProtocol
```

Comment:
Why the underscore here and the dash [here](https://github.com/rnystrom/GitHawk/pull/1122/files#diff-63c174961e625ddb07bf11ddd1035adbR23)?

Also, if there can be only one, why does this need to conform to `ListDiffable`?

Reply:
Can't use in IGListKit if not `ListDiffable` 😛